MycroftAI/mycroft-core
# Bump enclosure version

## Description
Bumps the used version of the mark-1 Arduino code to v1.4.2

- Fixes an old reference to ConfigurationManager
- Fixes handling of large images (on the Arduino side)

## How to test
Ensure that the mark-1 works as expected on the unstable repo branch and that the Orange eyes + UPDATE text disappears after the first restart.

## Contributor license agreement signed?
CLA [ Yes ]
